src/java.base/share/classes/java/net/doc-files/net-properties.html

```diff
@@ -124,31 +124,21 @@ <H2>Proxies</H2>
 		specified.</P>
                 <P>The default value excludes all common variations of the loopback address.</P>
 	</UL>
-	<LI><P>SOCKS<BR>This is another type of proxy. It allows for lower-level
-	type of tunneling since it works at the TCP level. In effect,
-	in the Java(tm) platform setting a SOCKS proxy server will result in
-	all TCP connections to go through that proxy, unless other proxies
-	are specified. If SOCKS is supported by a Java SE implementation, the
-	following properties will be used:</P>
+	<LI><P>SOCKS<BR>The SOCKS proxy enables a lower-level type of tunneling
+	because it works at the TCP level. Specifying a SOCKS proxy server
+	results in all java.net TCP connections going through that proxy server
```

Comment:
Maybe the "all java.net TCP connections" part should be reworked, to simply say that SOCKS proxy may not always be supported out-of-the-box depending on the type of connections. There you could cite NIO SocketChannel and java.net.http.HttpClient.